packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -239,8 +242,26 @@ export class MirrorNodeClient {
             buckets: [5, 10, 25, 50, 100, 250, 500, 1000, 2500, 5000, 10000, 20000, 30000] // ms (milliseconds)
         });
 
+        this.operatorAccount = process.env.OPERATOR_ID_MAIN?.trim() || '0.0.relayer';
+        const operatororBalanceCollect = async () => {
+            const account = await this.getAccount(this.operatorAccount);
+            this.operatorAccountGauge.labels({ 'accountId': this.operatorAccount }).set(account.balance.balance);
+        };
+
+        const metricGaugeName = 'rpc_relay_operator_balance';
+        register.removeSingleMetric(metricGaugeName);
+        this.operatorAccountGauge = new Gauge({
+            name: metricGaugeName,
+            help: 'Relay operator balance gauge',
+            labelNames: ['mode', 'type', 'accountId'],
+            registers: [register],
+            async collect() {
+                operatororBalanceCollect();
+            },
+        });
```

Comment:
Just one question though. 

Why are we having this here? and what is the diff with the one on `relay.ts` also on the one on the relay after getting the acount there is a check `?` before using the value, since it could lead to issues.
```const accountBalance = account.balance?.balance;```

https://github.com/hashgraph/hedera-json-rpc-relay/blob/1116-lru-refactor/packages/relay/src/lib/relay.ts#L101-L122

Reply:
I think the goal was to centralize mirrorNodeClient usage. In this case i should have removed it from `relay.ts`
I think i'll leave it there